frontend/src/scenes/insights/aggregationAxisFormat.ts

```diff
@@ -1,17 +1,21 @@
-import { LemonSelectOptions } from 'lib/components/LemonSelect'
-import { capitalizeFirstLetter, compactNumber, humanFriendlyDuration, percentage } from 'lib/utils'
+import { LemonSelectOption } from 'lib/components/LemonSelect'
+import { compactNumber, humanFriendlyDuration, percentage } from 'lib/utils'
 import { ChartDisplayType } from '~/types'
 
 const formats = ['numeric', 'duration', 'percentage'] as const
 export type AggregationAxisFormat = typeof formats[number]
 
-export const isAggregationAxisFormat = (candidate: unknown): candidate is AggregationAxisFormat =>
-    formats.includes(candidate as AggregationAxisFormat)
-
-export const aggregationAxisFormatSelectOptions = formats.reduce((target, format) => {
-    target[format as string] = { label: capitalizeFirstLetter(format as string) }
-    return target
-}, {} as LemonSelectOptions)
+export const aggregationAxisFormatSelectOptions: Record<AggregationAxisFormat, LemonSelectOption> = {
+    numeric: {
+        label: 'None',
+    },
+    duration: {
+        label: 'Duration (s)',
+    },
+    percentage: {
+        label: 'Percent (0-100)',
```

Comment:
We don't clamp this value to 100. So if you say something was "250" we'll print "250%"

Reply:
Ah, yeah, I think that's totally okay. The confusion might rather be between 1 being 1% or 100%.